iOS SDK Advertising ID documentation update (#698)
rudderlabs/rudderstack-docs: docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk/index.mdx

## Advertisement ID

> We have kept IDFA collection completely separate from the Core library so that the developer has better control over the same. You can pass the IDFA to `putAdvertisementId` method to set it under `context.device.advertisingId`.

Reviewer: ```suggestion
The IDFA collection is kept separately from the Core library so that the developer has better control over the same. You can pass the IDFA to `putAdvertisementId` method to set it under `context.device.advertisingId`.
```
```suggestion
We have kept IDFA collection completely separate from the Core library so that the developer has better control over the same. You can pass the IDFA to `putAdvertisementId` method to set it under `context.device.advertisingId`.
```
